Documentation/doc/Documentation/Installation.txt

```diff
@@ -508,6 +508,15 @@ and the \ref PkgRidges_3 packages.
 
 The \sc{Eigen} web site is <A HREF="http://eigen.tuxfamily.org">`http://eigen.tuxfamily.org`</A>.
 
+\subsection thirdpartyLapack Lapack
+
+\sc{Lapack} is a `Fortran` library for linear algebra.
+
+In \cgal, \sc{Lapack} is an alternative to \sc{Eigen} for singular value decomposition for the \ref
+PkgJet_fitting_3 and the \ref PkgRidges_3 packages.
+
+The \sc{Lapack} web site is <A HREF="http://www.netlib.org/lapack/">`http://www.netlib.org/lapack/`</A>.
+
```

Comment:
If those classes are not even tested by our testsuite, they should be considered as broken and should be removed.